src/MGA/Check/UnreachablePath.php

```diff
@@ -46,13 +49,29 @@ class UnreachablePath
         '.git/',
         '.git/refs/',
         '.gitignore',
+        '.idea',
         '.hg/',
+        '.htaccess.sample',
```

Comment:
This comes out of the box with Magento for `Enterprise_Staging` feature.  There wouldn't be anything interesting in here.